internal/operands/template-validator/reconcile.go

```diff
@@ -13,25 +13,32 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 
 	"kubevirt.io/ssp-operator/internal/common"
+	"kubevirt.io/ssp-operator/internal/operands"
 )
 
-func WatchTypes() []runtime.Object {
+type templateValidator struct{}
+
+func (t *templateValidator) AddWatchTypesToScheme(*runtime.Scheme) error {
+	return nil
```

Comment:
At first glance this looks like a mistake, I assume this function returns `nil` because no additional resources need to be added to the scheme, please add a comment to state that

Reply:
Correct. Resources from `k8s.io/api` are added by default. Ok I will add a comment.